### ISSUE TITLE
NVD3 Charts: Uncaught TypeError: Cannot read property 'each' of undefined

### ISSUE THREAD

**Issue by kusumakarb** [airbnb/superset#2329]
Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [x] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar

I installed superset using the ```pip install superset``` . Whenever I use any of the nvd3 charts the browser console displays the error:
```
Uncaught TypeError: Cannot read property 'each' of undefined     explore.0.15.4.entry.js:81
    at Object.t [as generate] (explore.0.15.4.entry.js:82)
    at e (explore.0.15.4.entry.js:78)
```

Even though the existing charts are being rendered properly, I'm trying to find out what is causing this.

### Superset version
"version": "0.15.4"

### Expected results
No error.

### Actual results
```
Uncaught TypeError: Cannot read property 'each' of undefined     explore.0.15.4.entry.js:81
    at Object.t [as generate] (explore.0.15.4.entry.js:82)
    at e (explore.0.15.4.entry.js:78)
```

### Steps to reproduce
1. Choose a table.
2. Choose any one of the nvd3 visualisations in the explorer view:
    NVD3TimeSeriesBarViz,
    DistributionBarViz,
    DistributionPieViz,
    NVD3TimeSeriesViz,
    NVD3TimeSeriesViz

3. Choose the parameters appropriately and query.
4. Check the browser console.

**Comment by mistercrunch** [comment]
Should be fixed in 0.17.0, reopen if you see it there

**Comment by kusumakarb** [comment]
It's not appearing in 0.17.0. Can you throw some light on why that was happening and how we fix it in the earlier versions? Thank you.
